internal/service/cloudprovideraccess/resource_cloud_provider_access_authorization.go

```diff
@@ -281,11 +281,9 @@ func authorizeRole(ctx context.Context, client *admin.APIClient, d *schema.Resou
 		roleID = targetRole.GetId()
 	}
 
-	var role *admin.CloudProviderAccessRole
 	var err error
-
 	for i := 0; i < 3; i++ {
-		role, _, err = client.CloudProviderAccessApi.AuthorizeCloudProviderAccessRole(ctx, projectID, roleID, req).Execute()
+		_, _, err = client.CloudProviderAccessApi.AuthorizeCloudProviderAccessRole(ctx, projectID, roleID, req).Execute()
```

Comment:
I don't think this change should be included

Reply:
reverted the change